Comp2023/src/main/java/frc/robot/subsystems/Elbow.java

```diff
@@ -73,14 +73,6 @@ public class Elbow extends SubsystemBase
   private StatorCurrentLimitConfiguration m_statorCurrentLimits = new StatorCurrentLimitConfiguration(true,
       ELConsts.kStatorCurrentLimit, ELConsts.kStatorTriggerCurrent, ELConsts.kStatorTriggerTime);
 
-  // Declare module variables
```

Comment:
We can keep the "Declare module variable" comment, since it addresses all of the variables below (not just the ones removed). Also, if you look at m_stickDeadband, it is like the others you removed. See if it can be changed.